File: CSharp/SpaceVIL/SpaceVIL/Decorations/Spacing.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SpaceVIL.Decorations
{
    /// <summary>
    /// A class that store vertical and horizontal spacing values of the item.
    /// </summary>
    public struct Spacing
    {
        /// <summary>
        /// Constructs a Spacing with strict horizontal and vertical spacing values.
        /// <para/> Default Spacing values is zeros.
        /// </summary>
        /// <param name="horizontal">Horizontal indent between items.</param>
        /// <param name="vertical">Vertical indent between items.</param>
        public Spacing(int horizontal = 0, int vertical = 0)
        {
            Horizontal = horizontal;
            Vertical = vertical;
        }
        /// <summary>
        /// Horizontal indent between items.
        /// </summary>
        public int Horizontal;
        /// <summary>
        /// Vertical indent between items.
        /// </summary>
        public int Vertical;
    }
}

File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/spacevil/Decorations/Spacing.java
package com.spvessel.spacevil.Decorations;

/**
 * A class that store vertical and horizontal spacing values of the item.
 */
public class Spacing {
    /**
     * Horizontal indent between items.
     */
    public int horizontal;

    /**
     * Vertical indent between items.
     */
    public int vertical;

    /**
     * Constructs a Spacing with default values (zero)
     */
    public Spacing() {
        horizontal = 0;
        vertical = 0;
    }

    /**
     * Constructs a Spacing with strict horizontal and vertical spacing values.
     * @param horizontal Horizontal indent between items.
     * @param vertical Vertical indent between items.
     */
    public Spacing(int horizontal, int vertical) {
        this.horizontal = horizontal;
        this.vertical = vertical;
    }
}